scripts/update-extension.sh

```diff
@@ -3,22 +3,21 @@
 source .env
 RELEASE_NAMESPACE="${RELEASE_NAMESPACE:-arc-osm-system}"
 EXTENSION_NAME="${EXTENSION_NAME:-osm}"
-API_VERSION="${API_VERSION:-2020-07-01-preview}"
+EXTENSION_TYPE="${EXTENSION_TYPE:-Microsoft.openservicemesh}"
 EXTENSION_SETTINGS=$1
+K8S_EXTENSION_VERSION="${K8S_EXTENSION_VERSION:-0.1PP.12}"
 
-export RESOURCEID=subscriptions/$SUBSCRIPTION/resourceGroups/$RESOURCEGROUP/providers/Microsoft.Kubernetes/connectedClusters/$CLUSTERNAME
+az account set --subscription=$SUBSCRIPTION > /dev/null 2>&1
+
+az extension remove --name k8s-extension
+
+git clone https://github.com/Azure/azure-arc-kubernetes-preview
 
-echo $RESOURCEID
+az extension add --source ./extensions/k8s_extension-$K8S_EXTENSION_VERSION-py2.py3-none-any.whl --yes
```

Comment:
Can you move lines 12-16 out of this script? This won't work for devs running this script locally

Reply:
Good catch, I can move this to be inside the pipeline.

Reply:
Users running this locally would have to have k8s-extension already installed before running the script.